tests/callbacks/test_early_stopping.py

```diff
@@ -228,7 +228,7 @@ def validation_epoch_end(self, outputs):
     early_stopping = EarlyStopping(
         monitor="abc", stopping_threshold=stopping_threshold, divergence_threshold=divergence_theshold
     )
-    trainer = Trainer(default_root_dir=tmpdir, callbacks=[early_stopping], overfit_batches=0.20, max_epochs=20)
+    trainer = Trainer(default_root_dir=tmpdir, callbacks=[early_stopping], max_epochs=20)
```

Comment:
I guess overfit_batches was used here to limit the batches for both train and val.
```suggestion
    trainer = Trainer(default_root_dir=tmpdir, callbacks=[early_stopping], limit_train_batches=0.2, limit_val_batches=0.2, max_epochs=20)
```